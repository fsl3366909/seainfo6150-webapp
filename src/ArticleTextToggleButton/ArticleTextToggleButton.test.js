import React from 'react';
import ArticleTextToggleButton from './ArticleTextToggleButton';

describe('ArticleList tests', () => {
	let isTextShowing = false;

	function onClick() {
		isTextShowing = !isTextShowing;
	}
	it('renders correctly for show less', () => {
		const { container } = render(<ArticleTextToggleButton buttonText="Show less" onClick={onClick} />);
		expect(container).toMatchSnapshot();
	});
	it('renders correctly for show more', () => {
		const { container } = render(<ArticleTextToggleButton buttonText="Show more" onClick={onClick} />);
		expect(container).toMatchSnapshot();
	});
});
