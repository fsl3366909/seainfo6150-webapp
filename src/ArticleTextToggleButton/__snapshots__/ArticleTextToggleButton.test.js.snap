// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList tests renders correctly for show less 1`] = `
<div>
  <button
    class="button"
  >
    Show less
  </button>
</div>
`;

exports[`ArticleList tests renders correctly for show more 1`] = `
<div>
  <button
    class="button"
  >
    Show more
  </button>
</div>
`;
